core/vision/src/main/java/com/arcrobotics/ftclib/vision/UGBasicHighGoalPipeline.java

```diff
@@ -73,43 +89,36 @@ public Mat processFrame(Mat input) {
 
         Imgproc.findContours(blueThreshold, blueContours, new Mat(), Imgproc.RETR_TREE, Imgproc.CHAIN_APPROX_SIMPLE);
         Imgproc.findContours(redThreshold, redContours, new Mat(), Imgproc.RETR_TREE, Imgproc.CHAIN_APPROX_SIMPLE);
+        blueContours = blueContours.stream().filter(i -> filterContours(i) && (((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) > 1) &&(((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) < 2)).collect(Collectors.toList());
+        redContours = redContours.stream().filter(i -> filterContours(i) && (((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) > 1) &&(((double) Imgproc.boundingRect(i).width / Imgproc.boundingRect(i).height) < 2)).collect(Collectors.toList());
 
+        Imgproc.drawContours(input, redContours, -1, new Scalar(255, 255, 0));
         if (blueContours.size() != 0) {
             biggestBlueContour = Collections.max(blueContours, new Comparator<MatOfPoint>() {
                 @Override
                 public int compare(MatOfPoint t0, MatOfPoint t1) {
-                    return Double.compare(Imgproc.contourArea(t0), Imgproc.contourArea(t1));
+                    return Double.compare(Imgproc.boundingRect(t0).width, Imgproc.boundingRect(t1).width);
                 }
             });
 
-
             blueRect = Imgproc.boundingRect(biggestBlueContour);
-            double aspectRatio = (double) blueRect.width / blueRect.height;
+            Imgproc.rectangle(input, blueRect, new Scalar(0, 0, 255), 3);
+            System.out.println((double) blueRect.width / (double) blueRect.height);
 
```

Comment:
Remove random whitespace between the `if` and `else`. It should be flush like so
```java
if (...) {
  // body
} else {
  // body
}
```